src/s3/activation.ts

```diff
@@ -30,6 +35,45 @@ export async function activate(extensionContext: vscode.ExtensionContext): Promi
         }),
         vscode.commands.registerCommand('aws.s3.uploadFile', async (node: S3BucketNode | S3FolderNode) => {
             await uploadFileCommand(node)
+        }),
+        vscode.commands.registerCommand('aws.s3.uploadFileToS3', async () => {
```

Comment:
Can we instead modify/enhance `uploadFileCommand()`? Why do we need a separate command?

Reply:
Just did

Reply:
the logic here should be in `uploadFileCommand()`